apps/engmanagement/src/pages/[slug].tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react'
+import {GetServerSideProps} from 'next'
+import {serialize} from 'next-mdx-remote/serialize'
+import {MDXRemote} from 'next-mdx-remote'
+import isEmpty from 'lodash/isEmpty'
+import find from 'lodash/find'
+import ArticleTemplate from 'templates/article'
+import ConvertkitSubscribeAndTagForm from 'components/convertkit-subscribe-and-tag'
+import groq from 'groq'
+import {sanityClient} from 'utils/sanity-client'
+import checkSubscriber from 'utils/check-subscriber'
+import {useNextSanityImage} from 'next-sanity-image'
+import Image from 'next/image'
+import {useConvertkit} from '@skillrecordings/convertkit'
+import {TheFutureOfRemoteWorkBackground} from 'components/backgrounds'
+
+export default function ExampleChapter({post, source, authorized}: any) {
```

Comment:
component name needs to be updated?

in almost every circumstance in our codebases we prefer a `const ComponentName = ()` approach over a named function like this. Are we introducing this pattern for a specific reason? I miss the typing of the props using this approach, but if it has an advantage I'm not aware of I'd like to hear ore.

Reply:
nah I simply missed this — updated, thanks!